types/index.d.ts

```diff
@@ -0,0 +1,7 @@
+declare function show(stream?: WritableStream): void;
```

Comment:
This file needs [TSDoc comments](https://github.com/sindresorhus/typescript-definition-style-guide#documentation).